jenkins_job_wrecker/cli.py

```diff
@@ -27,15 +28,17 @@
 
 
 def str_presenter(dumper, data):
-    if len(data.splitlines()) > 1:  # check for multiline string
-        # The dumper will not respect "style='|'" if it detects trailing
-        # whitespace on any line within the data. For scripts the trailing
-        # whitespace is not important.
-        lines = [l.rstrip() for l in data.splitlines()]
-        data = '\n'.join(lines)
+    lines = [l.rstrip() for l in data.splitlines()]
```

Comment:
Previously this was only done for multiline strings, not it also applies to 1-line strings - was this intended?